Source/Atomic/UI/UIWidget.h

```diff
@@ -250,10 +258,18 @@ class UIWidget : public Object, public tb::TBWidgetDelegate
     bool GetCaptured();
 
     void SetCapturing(bool capturing);
-    bool GetCapturing() { return widget_->GetCapturing(); }
+    bool GetCapturing();
 
     void InvokeShortcut(const String& shortcut);
 
+    bool GetShorten();
```

Comment:
Ok, GetShortened() would be good, does this have to be on UIWidget?  Doesn't it only make sense for text fields?